package.json

```diff
@@ -1,64 +1,44 @@
 {
-  "name": "richtypo",
-  "version": "0.0.0-development",
-  "description": "HTML typography enhancer for English and Russian texts",
-  "homepage": "https://github.com/sapegin/richtypo.js",
-  "author": {
-    "name": "Artem Sapegin",
-    "url": "http://sapegin.me/"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/sapegin/richtypo.js.git"
-  },
-  "bugs": {
-    "url": "https://github.com/sapegin/richtypo.js/issues"
+  "private": true,
+  "devDependencies": {
+    "@babel/cli": "^7.1.2",
+    "@babel/core": "^7.1.2",
+    "@babel/node": "^7.0.0",
+    "@babel/preset-env": "^7.1.0",
+    "babel-core": "^7.0.0-bridge",
+    "babel-jest": "^23.6.0",
+    "babel-plugin-transform-react-jsx": "^6.24.1",
+    "babel-watch": "https://github.com/monathur/babel-watch.git",
```

Comment:
Do you still use it?

Reply:
Sorry, not sure what you’re referring to?

Reply:
`babel-watch` should be updated soon to support babel 7 (see the issue here https://github.com/kmagiera/babel-watch/issues/75). I felt `babel-watch` was a better solution since it doesn't restart `babel-node` on every change.

Reply:
OK, let's keep it then.